.github/workflows/cd-app-deployment.yml

```diff
@@ -26,8 +26,8 @@ jobs:
     with:
       app_name: ${{ github.event.inputs.app_name }}
       image_tag: ${{ github.event.inputs.image_tag }}
-      resource_group_name: rgbeurokrattdev
-      k8s_cluster_name: bykaksdev
+      environment_name: dev
       namespace_name: applications
```

Comment:
supernit: I suppose we could make `namespace_name` have a default value of "applications" and not have to specify it everywhere we call the workflow. Would save a few lines and we would have the functionality to override if need be.

What do you think?

Reply:
that's a good point, but I recently read this security doc: 
https://github.com/buerokratt/Cross-functional-requirements/blob/main/docs/technical/security.md 
and there might be scope to put each app within their own namespace, so this might still need to be dynamically passed in, still open to discussion of course